docs/source/configuration/settings-reference.md

```diff
@@ -452,6 +452,10 @@ querystringSearchGet
     [See an explanation of character limits in URLs](https://stackoverflow.com/questions/417142/what-is-the-maximum-length-of-a-url-in-different-browsers/417184#417184).
     Please test this setting properly before enabling in a production site.
 
+openExternalLinkInNewTab 
+    A Boolean, `false` by default.
+    If openExternalLinkInNewTab is true and the URL is not an internal one, or if the target is '_blank', the openLinkInNewTab will be true, indicating that the link should be opened in a new tab.
```

Comment:
I'm not sure whether this is how it actually works. I'd have to research the implementation in the code to be certain. My suggestion is only for English grammar and MyST syntax.

```suggestion
    If either `openExternalLinkInNewTab` is `true` and the URL is external, or if `target` is `_blank`, then `openLinkInNewTab` will be `true`, indicating that the link, when clicked, will open in a new tab.
```